cyhy/core/geoloc.py

```diff
@@ -39,3 +40,14 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_cidr(self, cidr):
+        has_restricted = False
+        try:
+            response = self.__reader.city(str(cidr[0]))
```

Comment:
Looking at this again, this is only checking the first IP of an entire CIDR. I think that this method should take a single IP like the `lookup()` method, and the scripts should iterate over each IP in a CIDR to check.

Reply:
Great catch @mcdonnnj!

Reply:
Ah, you are right. I was thinking it would be enough that way but it should loop through them all to be sure. I will re-work the logic. My only thing is that I don't want to print out each and every IP that has an issue, as sometimes there will be /16, etc. So I will have it check all the IPs but only print the cidr if it is problematic. Less granularity of provided information, but it wont dump 65k IPs into the terminal. I will try and get that committed today for review.,